src/vs/workbench/contrib/files/browser/media/explorerviewlet.css

```diff
@@ -57,23 +57,6 @@
 	flex: 0; /* do not steal space when label is hidden because we are in edit mode */
 }
 
-.explorer-folders-view .pane-header .count {
```

Comment:
@lramos15 this seems like the bad rule here actually: I am not seeing how the dirty indicator of the "open editors" view would ever be in the `explorer-folders-view`, but maybe I am missing something. I have moved it into the CSS for open editors with this change and updated the rule.